### DISCUSSION TITLE
How do you calculate replaygain for APE format?

### DISCUSSION THREAD

**Discussion by laichiaheng** [DeaDBeeF-Player/deadbeef#2690] [General]
I hope that I'm not offending you, but I am curious how you do it on Linux.
I can't find any program do the same thing as DeaDBeeF.
SounKonverter: No.
metaflac: No.

**Comment by kpcee** [comment]
I'm not sure if I understand you correctly, but you can calculate replaygain for APE and other formats under Linux with e.g. [loudgain](https://github.com/Moonbase59/loudgain).

**Comment by laichiaheng** [comment]
> I'm not sure if I understand you correctly, but you can calculate replaygain for APE and other formats under Linux with e.g. [loudgain](https://github.com/Moonbase59/loudgain).

I found another one, r128gain.

**Comment by Oleksiy-Yakovenko** [comment]
deadbeef doesn't use external programs for replaygain. It uses [libebur128](https://github.com/jiixyj/libebur128) for calculating replaygain values.
